### PR TITLE
Fix/apollo client storybook

### PR DESCRIPTION
**Pull request by hdJerry**
In some cases, we get an "Apollo client not available" in the following stories:
Image
  - [GistsPanel.stories.ts](https://github.com/thisdot/starter.dev-github-showcases/blob/main/vue3-apollo-quasar/src/components/GistsPanel/GistsPanel.stories.ts "GistsPanel.stories.ts")
  - [NavHeader.stories.ts](https://github.com/thisdot/starter.dev-github-showcases/blob/main/vue3-apollo-quasar/src/components/NavHeader/NavHeader.stories.ts "NavHeader.stories.ts")

# Acceptance
- Add https://storybook.js.org/addons/storybook-addon-apollo-client to the project
- Configure stories that rely on apollo to use this addon
- Ensure all data used in the stories are mocked and don't use real data
- If this is a good addon to have, inform Dustin or Jeremy so we can add it to the kit officially

## Fix
Adding [Storybook Addon Apollo Client](https://storybook.js.org/addons/storybook-addon-apollo-client) to the project didn't seem to work as there is an issue with it after checking their repo [Vue/Nuxt – TypeError: Cannot add property components, object is not extensible](https://github.com/lifeiscontent/storybook-addon-apollo-client/issues/72). 

So, I opted for [MSW Storybook Addon
](https://github.com/mswjs/msw-storybook-addon)

Screenshot 2022-08-30 at 14 11 48

Screenshot 2022-08-30 at 13 19 15

### CODE REVIEW

**Comment by AllanJeremy** [comment]
CI fails @hdJerry ~ kindly resolve that first

File: vue3-apollo-quasar/.storybook/preview.js [hunk 1]
```diff
@@ -10,11 +10,23 @@ import 'quasar/dist/quasar.css';
...
 import { createPinia } from 'pinia';
+import { initialize, mswDecorator } from 'msw-storybook-addon';
+
+import { setupGraphQL } from '@/init';
 
 const pinia = createPinia();
 
 app.use(Quasar, {}).use(pinia);
 
+// Initialize MSW
```
Comment (AllanJeremy) [inline vue3-apollo-quasar/.storybook/preview.js:21]: Please add more useful info to this comment. What's MSW? Why are we using it?

```
MSW allows us to mock service workers...
```

**Review by AllanJeremy — CHANGES_REQUESTED** [review]
Minor request

File: vue3-apollo-quasar/.storybook/preview.js (hunk 1 above)
Comment (hdJerry) [reply vue3-apollo-quasar/.storybook/preview.js:21]: DOn't you think that will be cooler in the readme?
@AllanJeremy

Comment (AllanJeremy) [reply vue3-apollo-quasar/.storybook/preview.js:21]: Added this to [the documentation PR](https://github.com/thisdot/starter.dev-github-showcases/pull/563)

**Review by AllanJeremy — APPROVED** [review]
🚢

**Pull request merged**